src/alto-ui/Dialog/__tests__/Dialog.test.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Dialog from '../Dialog';
+
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return props => {
+    return <button className="CloseButton" onClick={() => props.onClick()} />;
+  };
+});
+
+describe('Dialog', () => {
+
+  const defaultProps = {
+    id: 'string',
+    // className: 'string',
+    // open: true,
+    // title: 'string',
+    // onClose: jest.fn(),
+    // buttons: [{}, {}],
+    // closeFocusTargetId: 'string',
+    // inert: true,
+    // a11yLabelClose: 'string',
+  }
+
+  const getWrapper = ({ ...props }) => shallow(<Dialog {...defaultProps} {...props} />);
+
+  it('is Dialog',() => {
+    const wrapper = getWrapper({
+      id: 'key'
+    });
+    expect(wrapper.find('.Dialog__overlay').exists()).toBe(true);
+  })
+
+  it('should render div with Dialog className',() => {
+    const props = {
+      open: true,
+    }
+
+    const wrapper = getWrapper(props);
+    expect(wrapper.find('.Dialog').exists()).toBe(true);
+  })
+
+  it('should render header with Dialog__header className',() => {
```

Comment:
should has header element

Reply:
Refactored to `it(`should has element with Dialog__header className`)`